selfdrive/loggerd/loggerd.cc

```diff
@@ -658,35 +659,26 @@ int main(int argc, char** argv) {
         if (msg == NULL){
           break;
         }
-
-        uint8_t* data = (uint8_t*)msg->getData();
-        size_t len = msg->getSize();
-
+        MessageReader amsg(msg);
```

Comment:
This seems slower, we should only need to create this for frames.

Reply:
I have added getData(), getSize() into MessageReader to avoid declare Message* here to make pattern consistent with others